fluence-faas/src/faas.rs

```diff
@@ -45,6 +50,7 @@ impl<'a> ModulesLoadStrategy<'a> {
         match self {
             ModulesLoadStrategy::All => true,
             ModulesLoadStrategy::Named(set) => set.contains(module),
+            ModulesLoadStrategy::WasmOnly => module.ends_with(".wasm"),
```

Comment:
Suggest to wrap it with `Path` (it costs roughtly nothing) and call `extenstion()` method. It is less error-prone way to check extenstion, moreover it works on different OS.

In current version files with name `.wasm` would pass this check and then 
```rust
ModulesLoadStrategy::WasmOnly => module
                .strip_suffix(".wasm")
                .map(|s| s.to_string())
                .unwrap_or(module),
```

strips entire file name and FCE loads module with empty name, and we can't call it from REPL.